chips/stm32h7-nonredundant/openocd.cfg

```diff
@@ -0,0 +1,43 @@
+gdb_port 3343
+telnet_port 4453
+
+source [find interface/stlink.cfg]
+source [find target/stm32h7x_dual_bank.cfg]
```

Comment:
Should this be the normal `stm32h7x.cfg`?

Reply:
No, that config is for any part with the dual-bank flash controller.